plus.js

```diff
@@ -109,13 +117,32 @@ document.arrive(".shaka-volume-bar-container", function() {
         // play/pause button
         play_button = "<button class='material-icons' id='mpp-play' aria-label='Play/Pause' title='Play/Pause'>play_arrow</button>";
         document.getElementsByClassName("shaka-current-time")[0].insertAdjacentHTML("beforebegin", play_button);
-        document.getElementById("mpp-play").addEventListener('click', function() {
-            if (vid.paused) {
-                vid.play();
+
+         // Workaround to stop the spacebar "clicking" as a generic keyevent causing a double play/pause
```

Comment:
What is this about?

Reply:
Good Question - I had not rebased my branch, so I think git had a fit.